Makefile

```diff
@@ -15,26 +15,32 @@
 ##########################################################
 
 GAFFER_VERSION=0.4.4
+VERSION=0.4.4c
+ACCUMULO_REPOSITORY=cybermaggedon/accumulo-gaffer
+WILDFLY_REPOSITORY=cybermaggedon/wildfly-gaffer
```

Comment:
We will probably want to change these to our GCHQ docker repository. I have created a develop branch in this gaffer-docker repository. 

Please could you change the pull request to merge it into the new develop branch? This will allow us to make minor changes and push up a release to docker hub before we merge it into master.